compiler/src/main/java/com/raizlabs/android/dbflow/processor/definition/TableDefinition.java

```diff
@@ -198,18 +198,18 @@ public void write(JavaWriter javaWriter) throws IOException {
 
                     if (columnDefinition.columnType == Column.FOREIGN_KEY) {
                         for (ForeignKeyReference reference : columnDefinition.foreignKeyReferences) {
-                            columnNames.add(reference.columnName());
+                            columnNames.add(QueryBuilder.quote(reference.columnName()));
```

Comment:
This append quotes to column names. Which later are appended to query using appendList. Should i add appendListQuoted Method and use it here?